core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-section/template-builder-section.component.scss

```diff
@@ -1,12 +1,23 @@
 @use "variables" as *;
 
 :host {
-    display: flex;
-    justify-content: center;
+    display: grid;
     align-items: center;
+    grid-template-columns: repeat(3, 1fr);
     background: $color-palette-primary-200;
     height: 5rem;
     color: $color-palette-primary;
     font-size: $font-size-xl;
     border-radius: $spacing-1;
+    padding: 0 $spacing-0;
+
+    .template-builder-section__content {
+        text-align: center;
+        grid-column: 2/2;
+    }
+
+    p-button {
+        grid-column: 3/3;
+        justify-self: flex-end;
+    }
```

Comment:
This `p-button`should not take space in the centering of the content... absolute position here.

Reply:
I tried to do that, but it broke the gridstack grid, I can give it another try